src/Files.Uwp/Helpers/LayoutPreferences/LayoutPrefsDb.cs

```diff
@@ -0,0 +1,143 @@
+﻿using LiteDB;
+using System;
+using System.Linq;
+using Files.Shared.Extensions;
+
+namespace Files.Uwp.Helpers.LayoutPreferences
+{
+    public class LayoutPrefsDb : IDisposable
+    {
+        private readonly LiteDatabase db;
+
+        public LayoutPrefsDb(string connection, bool shared = false)
+        {
+            db = new LiteDatabase(new ConnectionString(connection)
```

Comment:
Out of curiosity, was there a specific reason to use LiteDB?

Reply:
We use it for file tags, where it's needed to allow concurrent access to the DB from the UWP and FulltrustProcess processes. Since the dependency is there, I'm using it for layout prefs as well (I like the API more than e.g the SQLite one).